src/common/utils/customErrors/apiError.ts

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import CustomError from "./customError";
+
+class APITokenError extends CustomError {
+  constructor(message: string, data: any, err: any) {
+    super({});
```

Comment:
@pichu07 Use super to access all the parent class parameters. Change `super({})`

Reply:
@pichu07 You can only have `errorType` = this.constructor.name in each error class. Rest of the parameters can go into  super.